packages/contentful--app-scripts/src/analytics.ts

```diff
@@ -7,10 +7,10 @@ const SEGMENT_WRITE_KEY = 'IzCq3j4dQlTAgLdMykRW9oBHQKUy1xMm';
  *
  * @param {object} properties tracking properties
  * @param {string} properties.command triggered command e.g create-app-definition, upload, etc.
- * @param {boolean} properties.ci value if --ci flag has been set
+ * @param {string} properties.ci value if --ci flag has been set
```

Comment:
String?

Reply:
Yes, we pass string here, for example:
https://github.com/contentful/create-contentful-app/pull/1634/files#diff-93c382db7222a985fff313bc92ddd886018770fe7872c55bbf06dce16b6fff52R72

Reply:
Feels quite wrong as it's a boolean flag 🤔

Reply:
Schema accepts string, I guess:
https://github.com/contentful/create-contentful-app/pull/1524

Reply:
But we can convert it to string inside of `track`, I believe.